src/data/matrix/basic.lean

```diff
@@ -176,6 +177,17 @@ by simp [bit1_val, h]
 
 end numeral
 
+section add_group
+
+variables [add_group α]
+
+lemma neg_diagonal (d : n → α) : -diagonal d = diagonal (-d) :=
```

Comment:
`@[simp]`?

Reply:
Actually this lemma (with `@[simp]` attribute) already exists and is called `diagonal_neg` so I have just removed mine. 🤦‍♂️